### PR TITLE
pitr: add known issues

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/pitr-known-issues.md [hunk 1]
```diff
@@ -15,8 +15,11 @@ Issue 链接：[#36648](https://github.com/pingcap/tidb/issues/36648)
 
 执行 PITR 恢复时遇到 BR OOM 问题，从以下几点考虑：
 
-- PITR 恢复出现 OOM，是由于待恢复的日志区间过大。
-    - 一般建议恢复的日志区间不超过 2 天，最长不超过一周。即在 PITR 备份过过程中至少 2 天或一周至少做一次全量备份操作。
+- PITR 恢复出现 OOM，是由于待恢复的索引日志数据过多，以下是两个典型场景。
+    - 恢复日志区间过大
+    	- 建议恢复的日志区间不超过 2 天，最长不超过一周。即在 PITR 备份过过程中至少 2 天或一周至少做一次全量备份操作。
+    - 备份日志期间，存在长时间大量写入， 比如单 KV 节点写入 ops 超过 50k/s。
+        - 这个场景一般出现在初始化集群全量准备数据阶段，建议增加全量备份的频率，比如在导入完成后，进行一次全量备份，缩短全量备份 snapshot 与恢复目标时间点的时间差。
```
Comment (IANTHEREAL) [inline br/pitr-known-issues.md:22]:
```suggestion
        - 这个场景一般出现在初始化集群全量准备数据阶段，建议在导入完成后进行一次全量备份，使用该备份进行恢复。
```

File: br/pitr-known-issues.md [hunk 2]
```diff
@@ -53,3 +56,17 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
 Issue 链接：[#13126](https://github.com/tikv/tikv/issues/13126)
 
 在集群出现网络分区故障后，备份任务难以继续备份日志，并且在超过一定的重试时间后，任务会被置为 `ERROR` 状态。此时备份任务已经停止，需要手动执行 `br log resume` 命令来恢复日志备份任务。
+
+## 集群监控显示数据增量 385 GB，日志备份实际消耗存储大约 1 TB.
+
+Issue 链接：[#13306](https://github.com/tikv/tikv/issues/13306)
+
+这是由于集群监控显示的是 rocksdb 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致。而日志备份没有选择使用 rocksdb 生成 SST 文件的方式存储数据的原因是，日志备份期间生成的数据会遇到区间范围过大，但是实际区间内容较少的情况，这个情况下，通过 ingest SST 的方式恢复数据，效果会大打折扣。
```
Comment (IANTHEREAL) [inline br/pitr-known-issues.md:64]:
```suggestion
这是由于集群监控显示的是 rocksdb 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致，大约是 x ~ y 倍。而日志备份没有选择使用 rocksdb 生成 SST 文件的方式存储数据的原因是，日志备份期间生成的数据会遇到区间范围过大，但是实际区间内容较少的情况，这个情况下，通过 ingest SST 的方式恢复数据，效果会大打折扣。
```

Comment (IANTHEREAL) [reply br/pitr-known-issues.md:64]: 帮忙补充一下数字

File: br/pitr-known-issues.md [hunk 3]
```diff
@@ -53,3 +56,17 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
...
+
+## 执行 PITR 恢复时遇到 execute over region id 报错
+
+Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
+
+该场景发生在没有全量备份作为基础时，仅通过日志数据来恢复全量导入的集群初始化数据，经过测试发现，当存在长时间(24h)大量热点写入，且平均单台 TiKV 节点写入 ops > 50k/s ，那么有几率会遇到这个情况。
```
Comment (IANTHEREAL) [inline br/pitr-known-issues.md:70]:
```suggestion
该场景发生在通过日志数据来恢复全量导入的集群初始化数据，经过测试发现，当存在长时间(24h)大量热点写入，且平均单台 TiKV 节点写入 ops > 50k/s ，那么有几率会遇到这个情况。
```

File: br/pitr-known-issues.md (hunk 2 above)
Comment (3pointer) [reply br/pitr-known-issues.md]: done

**Comment by IANTHEREAL** [comment]
LGTM

**Review by IANTHEREAL — APPROVED** [review]

File: br/pitr-known-issues.md [hunk 4]
```diff
@@ -53,3 +56,17 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
...
 
 在集群出现网络分区故障后，备份任务难以继续备份日志，并且在超过一定的重试时间后，任务会被置为 `ERROR` 状态。此时备份任务已经停止，需要手动执行 `br log resume` 命令来恢复日志备份任务。
+
+## 集群监控显示数据增量 385 GB，日志备份实际消耗存储大约 1 TB
+
+Issue 链接：[#13306](https://github.com/tikv/tikv/issues/13306)
+
+这是由于集群监控显示的是 rocksdb 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致，大约是 2～3 倍之间。而日志备份没有选择使用 rocksdb 生成 SST 文件的方式存储数据的原因是，日志备份期间生成的数据会遇到区间范围过大，但是实际区间内容较少的情况，这个情况下，通过 ingest SST 的方式恢复数据，效果会大打折扣。
+
+## 执行 PITR 恢复时遇到 execute over region id 报错
```
Comment (joccau) [inline br/pitr-known-issues.md:66]:
```suggestion
## 执行 PITR 恢复时遇到 `execute over region id` 报错
```

**Review by fubinzh — APPROVED** [review]
LGTM

File: br/pitr-known-issues.md [hunk 5]
```diff
@@ -53,3 +56,23 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
...
+
+Issue 链接：[#13306](https://github.com/tikv/tikv/issues/13306)
+
+这是由于集群监控显示的是 rocksdb 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致，大约是 2～3 倍之间。而日志备份没有选择使用 rocksdb 生成 SST 文件的方式存储数据的原因是，日志备份期间生成的数据会遇到区间范围过大，但是实际区间内容较少的情况，这个情况下，通过 ingest SST 的方式恢复数据，效果会大打折扣。
+
+## 执行 PITR 恢复时遇到 `execute over region id` 报错
+
+Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
+
+该场景发生在通过日志数据来恢复全量导入的集群初始化数据，经过测试发现，当存在长时间(24h)大量热点写入，且平均单台 TiKV 节点写入 ops > 50k/s(可以通过 grafana 中 `TiKV-Details` -> `Backup Log` -> `Handle Event Rate` 确认该数值)，那么有几率会遇到这个情况。
```
Comment (joccau) [inline br/pitr-known-issues.md:70]:
```suggestion
该场景发生在全量数据导入时开启了日志备份，并使用 PiTR 恢复全量导入时间段的日志。经过测试发现，当存在长时间(24h)大量热点写入，且平均单台 TiKV 节点写入 ops > 50k/s(可以通过 grafana 中 `TiKV-Details` -> `Backup Log` -> `Handle Event Rate` 确认该数值)，那么有几率会遇到这个情况。
```

Comment (3pointer) [reply br/pitr-known-issues.md:70]: I feel like both description is ok? WDYT? cc @IANTHEREAL

**Review by joccau — APPROVED** [review]
rest LGTM

File: br/pitr-known-issues.md [hunk 6]
```diff
@@ -15,8 +15,11 @@ Issue 链接：[#36648](https://github.com/pingcap/tidb/issues/36648)
 
 执行 PITR 恢复时遇到 BR OOM 问题，从以下几点考虑：
 
-- PITR 恢复出现 OOM，是由于待恢复的日志区间过大。
-    - 一般建议恢复的日志区间不超过 2 天，最长不超过一周。即在 PITR 备份过过程中至少 2 天或一周至少做一次全量备份操作。
+- PITR 恢复出现 OOM，是由于待恢复的索引日志数据过多，以下是两个典型场景。
+    - 恢复日志区间过大
+        - 建议恢复的日志区间不超过 2 天，最长不超过一周。即在 PITR 备份过过程中至少 2 天或一周至少做一次全量备份操作。
```
Comment (ran-huang) [inline br/pitr-known-issues.md:20]: 1. 文档里有很多地方交替使用“全量备份”和“快照备份”，建议统一为“快照备份”。
2. 这里的意思是至少两天做一次，还是至少一周做一次？区别有点大

```suggestion
        - 建议恢复的日志区间不超过 2 天，最长不超过一周。即在 PITR 备份过程中，至少每 2 天或每 1 周做一次快照备份操作。
```

File: br/pitr-known-issues.md [hunk 7]
```diff
@@ -15,8 +15,11 @@ Issue 链接：[#36648](https://github.com/pingcap/tidb/issues/36648)
...
+    - 备份日志期间，存在长时间大量写入。
+        - 这个场景一般出现在初始化集群全量准备数据阶段，建议在导入完成后进行一次全量备份，使用该备份进行恢复。
```
Comment (ran-huang) [inline br/pitr-known-issues.md:22]:
```suggestion
        - 长时间大量写入的场景一般出现在初始化集群全量导入数据的阶段。建议在导入完成后进行一次快照备份，使用该备份进行恢复。
```

File: br/pitr-known-issues.md [hunk 8]
```diff
@@ -53,3 +56,23 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
 Issue 链接：[#13126](https://github.com/tikv/tikv/issues/13126)
 
 在集群出现网络分区故障后，备份任务难以继续备份日志，并且在超过一定的重试时间后，任务会被置为 `ERROR` 状态。此时备份任务已经停止，需要手动执行 `br log resume` 命令来恢复日志备份任务。
+
+## 集群监控显示数据增量 385 GB，日志备份实际消耗存储大约 1 TB
```
Comment (ran-huang) [inline br/pitr-known-issues.md:60]:
```suggestion
## 日志备份实际消耗的存储是集群监控显示的数据增量的 2~3 倍
```

File: br/pitr-known-issues.md [hunk 9]
```diff
@@ -53,3 +56,23 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
...
+
+Issue 链接：[#13306](https://github.com/tikv/tikv/issues/13306)
+
+这是由于集群监控显示的是 rocksdb 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致，大约是 2～3 倍之间。而日志备份没有选择使用 rocksdb 生成 SST 文件的方式存储数据的原因是，日志备份期间生成的数据会遇到区间范围过大，但是实际区间内容较少的情况，这个情况下，通过 ingest SST 的方式恢复数据，效果会大打折扣。
```
Comment (ran-huang) [inline br/pitr-known-issues.md:64]: “效果会大打折扣”有点模糊，可以具体一点吗？是指恢复数据的速度很慢？
```suggestion
这是由于集群监控显示的是 RocksDB 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致，大约是 2~3 倍之间。日志备份没有使用 RocksDB 生成 SST 文件的方式存储数据，是因为日志备份期间生成的数据会遇到区间范围过大而实际区间内容较少的情况。这个情况下，通过 ingest SST 的方式恢复数据，效果会大打折扣。
```

File: br/pitr-known-issues.md [hunk 10]
```diff
@@ -53,3 +56,23 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
...
+
+Issue 链接：[#13306](https://github.com/tikv/tikv/issues/13306)
+
+这是由于集群监控显示的是 RocksDB 压缩后的数据，而日志备份使用的是自定义的编码方式存储 KV 数据，因此造成压缩比率不一致，大约是 2~3 倍之间。日志备份没有使用 RocksDB 生成 SST 文件的方式存储数据，是因为日志备份期间生成的数据会遇到区间范围过大而实际区间内容较少的情况。这个情况下，通过 ingest SST 的方式恢复数据，并不能有效提升恢复性能。
+
+## 执行 PITR 恢复时遇到 `execute over region id` 报错
+
+Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
+
+该场景发生在全量数据导入时开启了日志备份，并使用 PiTR 恢复全量导入时间段的日志。经过测试发现，当存在长时间(24h)大量热点写入，且平均单台 TiKV 节点写入 ops > 50k/s(可以通过 grafana 中 `TiKV-Details` -> `Backup Log` -> `Handle Event Rate` 确认该数值)，那么有几率会遇到这个情况。
```
Comment (ran-huang) [inline br/pitr-known-issues.md:70]:
```suggestion
该场景发生在全量数据导入时开启了日志备份，并使用 PiTR 恢复全量导入时间段的日志。经过测试发现，当存在长时间（24 小时）大量热点写入，且平均单台 TiKV 节点写入 OPS > 50k/s（可以通过 Grafana 中 **TiKV-Details** -> **Backup Log** -> **Handle Event Rate** 确认该数值），那么有几率会遇到这个情况。
```

File: br/pitr-known-issues.md [hunk 11]
```diff
@@ -53,3 +56,23 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
...
+
+- 当前版本中建议在集群初始化后，进行一次有效快照备份，并且以此作为基础进行 PITR 恢复。
+
+## 当存在大事务的时候，事务的提交时间会影响日志备份 checkpoint lag
+
+Issue 链接：[#13304](https://github.com/pingcap/tidb/issues/13304)
```
Comment (ran-huang) [inline br/pitr-known-issues.md:76]: issue 链接是不是写错了，这是一个已经合并的 PR

File: br/pitr-known-issues.md [hunk 12]
```diff
@@ -53,3 +56,23 @@ Issue 链接：[#13030](https://github.com/tikv/tikv/issues/13030)
...
+
+当场景中有大事务时，日志 checkpoint lag 会增加接近大事务的提交时间。
```
Comment (ran-huang) [inline br/pitr-known-issues.md:78]: “接近大事务的提交时间”是指什么

Comment (3pointer) [reply br/pitr-known-issues.md:78]: 比如，事务提交花了20分钟才完成，那么 checkpoint lag 在事务提交前都不会更新。也会看到 20 分钟的 lag

File: br/pitr-known-issues.md (hunk 11 above)
Comment (3pointer) [reply br/pitr-known-issues.md]: 修复了，这个是 tikv 的 issue，我贴错了

File: br/pitr-known-issues.md (hunk 12 above)
Comment (ran-huang) [reply br/pitr-known-issues.md:78]: 这样解释一下可以吗？
```suggestion
当场景中有大事务时，日志 checkpoint lag 在事务提交前都不会更新，因此会增加一段接近于大事务提交时长的时间。
```

Comment (3pointer) [reply br/pitr-known-issues.md]: 感觉可以

**Review by ran-huang — APPROVED** [review]

**Review by IANTHEREAL — APPROVED** [review]

**Pull request merged**